packages/rn-tester/Podfile

```diff
@@ -3,9 +3,8 @@ require_relative '../../scripts/react_native_pods'
 source 'https://cdn.cocoapods.org/'
 platform :ios, '11.0'
 
-# TODO(macOS GH#214)
-# Otherwise duplicate UUIDs are being generated between the iOS and macOS targets.
-# FIXME: https://github.com/microsoft/react-native/issues/210
+# Temporary solution to suppress duplicated GUID error.
+# Can be removed once we move to generate files outside pod install.
```

Comment:
Missing TODO tag?

Reply:
No TODO tag needed. This comment still exists in RN Core.

https://github.com/facebook/react-native/commit/f7e4c07c84b636fc33c64b434964c8a64c43438f#diff-22b542fd5caa8059ff2b6ca338317deadaa006db2b45cd5b060dc3a12c880966